kv/kv.go

```diff
@@ -372,6 +374,11 @@ type Request struct {
 	RequestSource util.RequestSource
 }
 
+// ClosestReadChecker is used to check whether a copTask can be executed by a follower
+// the input is the number of cop tasks, return true means this copTask can be send to
+// a follower.
+type ClosestReadChecker func(*Request, int) bool
```

Comment:
The name `XXXChecker` seems that it won't modify the input Request and just do some checks.
Please add some comments to notify others that this function may modify the input Request if you want to use this name.